velox/serializers/PrestoSerializer.h

```diff
@@ -20,6 +20,17 @@
 namespace facebook::velox::serializer::presto {
 class PrestoVectorSerde : public VectorSerde {
  public:
+  // Input options that the serializer recognizes.
+  struct PrestoOptions : public VectorSerde::Options {
+    PrestoOptions(bool useLosslessTimestamp)
+        : useLosslessTimestamp_(useLosslessTimestamp) {}
+    // Currently presto only supports millisecond precision and the serializer
+    // converts velox native timestamp to that resulting in loss of precision.
+    // This option allows it to serialize with nanosecond precision and is
+    // currently used for spilling. Is false by default.
+    bool useLosslessTimestamp_{false};
```

Comment:
Public field does not need postfix with `_`

Reply:
done